doozer

```diff
@@ -79,6 +103,8 @@ def print_version(ctx, param, value):
               help='If a base image is not included, lookup latest FROM tag for parent. Implies --ignore-missing-base')
 @click.option("--quiet", "-q", default=False, is_flag=True, help="Suppress non-critical output")
 @click.option('--debug', default=False, is_flag=True, help='Show debug output on console.')
+@click.option("--source", metavar="ALIAS PATH", nargs=2, multiple=True,
+              help="Associate a path with a given source alias.  [multiple]")
```

Comment:
... isn't this a duplicate of the next 2 lines?

Reply:
Oops... Yeah. Will remove.